releases/release-1.32/release-notes/maps/pr-125923-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 125923
+releasenote:
+  text: 'Fixed a bug where the kubelet ephemerally failed with `failed to initialize
+    top level QOS containers: root container [kubepods] doesn''t exist`, due to the
+    cpuset cgroup being deleted on v2 with systemd cgroup manager.'
```

Does the extra `''` break the yaml?

```suggestion
  text: 'Fixed a bug where the kubelet ephemerally failed with `failed to initialize
    top level QOS containers: root container [kubepods] doesn't exist`, due to the
    cpuset cgroup being deleted on cgroup v2 with systemd cgroup manager.'
```